src/router/hooks/beforeEachRoute.js

```diff
@@ -60,6 +60,8 @@ export default async function beforeEachRoute(to, __, next) {
     }
   }
 
+  const shouldRedirectToBilling = accountStore.isReviewPaymentRequired && isNotBillingRoute
```

Comment:
poderia está em um arquivo separado essa regra.

Reply:
Na verdade, essa variável foi criada com esse objetivo específico. Separar essa lógica em outro arquivo pode dificultar o entendimento do fluxo de rotas. Se ela ficar mais complexa ou for reutilizada, aí sim vale separar. Por enquanto, acho melhor manter onde está para facilitar a leitura.